### PR TITLE
version Command

### PR DESCRIPTION
**Pull request by hfuss**
Provides a command for listing the `ff` build version. Created an internal package for version so that it was accessible to the `versionCmd` in the `cmd` package.

Alot of folks on the web use `main.Version` in their `ldflags` but we can't access that outside the main package.

### CODE REVIEW

File: internal/version/version.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+var Version = "v0.0.1"
```
Comment (hfuss) [inline internal/version/version.go:19]: has to be a var in order for `ldflags` to be able to set it

**Comment by hfuss** [comment]
Demo:

```shell
❯ ./ff/ff version -s
canary

❯ ./ff/ff version -o json
{
  "Version": "canary",
  "Commit": "2d68b31",
  "Date": "2022-01-07T03:12:46Z"
}

❯ ./ff/ff version -o yaml
Version: canary
Commit: 2d68b31
Date: "2022-01-07T03:12:46Z"

❯ ./ff/ff version --help
Prints the version info of the CLI binary

Usage:
  ff version [flags]

Flags:
  -h, --help            help for version
  -o, --output string   output format ("yaml"|"json") (default "json")
  -s, --short           print only the version

Global Flags:
      --ansi string   control when to print ANSI control characters ("never"|"always"|"auto") (default "auto")
  -v, --verbose       verbose log output
```

```

File: cmd/prompt.go [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
-			str = strings.ToLower(strings.TrimSpace(str))
-			if str == "y" || str == "yes" {
-				return nil
-			} else {
-				return fmt.Errorf("confirmation declined with response: '%s'", str)
-			}
-		}
-	}
-}
+// Copyright © 2022 Kaleido, Inc.
```
Comment (hfuss) [inline cmd/prompt.go:1]: result of gofmt

**Review by peterbroadhurst — APPROVED** [review]
Thanks @hfuss - this looks great

**Pull request merged**